src/components/common/RecipientsTable/RecipientTable.js

```diff
@@ -25,6 +18,48 @@ import {
   getAllHouseholdAction,
 } from '../../../state/actions';
 
+const genders = [
```

Comment:
Ideally we would reference these items instead of hardcoding them. This will need to be done in a future push. Please be sure to document this issue.